src/hotspot/cpu/riscv/vm_version_riscv.cpp

```diff
@@ -248,6 +248,11 @@ void VM_Version::initialize() {
     warning("Block zeroing is not available");
     FLAG_SET_DEFAULT(UseBlockZeroing, false);
   }
+  if (UseRVV) {
```

Comment:
What happens if someone enables `+UseChaCha20Intrinsics` without `+UseRVV`? Maybe check if `-UseRVV` and `+UseChaCha20Intrinsics` and disable it?

Reply:
Nice catch! Thanks Tony!